tools/test-staking.ts

```diff
@@ -92,7 +92,7 @@ async function test() {
         unsub();
       }
     });
-  await wait(50000);
+  await wait(80000);
```

Comment:
I wonder why this change was necessary. Why does the block take more time now vs before?

Reply:
I'm not asking _you_, just in general wondering.

Reply:
> have to wait two sessions